yarn.lock

```diff
@@ -7954,10 +7937,10 @@ marked-terminal@^4.1.0:
     node-emoji "^1.10.0"
     supports-hyperlinks "^2.1.0"
 
-marked@^4.0.12:
-  version "4.0.14"
-  resolved "https://registry.yarnpkg.com/marked/-/marked-4.0.14.tgz#7a3a5fa5c80580bac78c1ed2e3b84d7bd6fc3870"
-  integrity sha512-HL5sSPE/LP6U9qKgngIIPTthuxC0jrfxpYMZ3LdGDD3vTnLs59m2Z7r6+LNDR3ToqEQdkKd6YaaEfJhodJmijQ==
+marked@^4.0.12, marked@^4.0.14:
+  version "4.1.1"
```

Comment:
Do we need this to be upgraded?

Reply:
I used the latest version used somewhere else in our source code. We must be consistent.
Anyway, the change in `package.js`this file is reverted.